documentation/asciidoc/topics/proc_connecting_cli.adoc

```diff
@@ -27,12 +34,25 @@ $ {kubectl_exec}
 However using the CLI in this way consumes memory allocated to the container, which can lead to out of memory exceptions.
 ====
 
+.Procedure
+
 . Create a CLI connection to your {brandname} cluster.
 +
+ifdef::downstream[]
+.Using the server distribution
+endif::downstream[]
 [source,bash,options="nowrap",subs=attributes+]
 ----
 $ bin/cli.sh -c https://$SERVICE_HOSTNAME:$PORT --trustall
 ----
+ifdef::downstream[]
++
+.Using the native CLI
+[source,bash,options="nowrap",subs=attributes+]
+----
+$ ./datagrid-cli -c https://$SERVICE_HOSTNAME:$PORT
```

Comment:
I don't think we can merge this until https://github.com/infinispan/infinispan-quarkus/pull/61 is merged and available downstrream. We may have missed the boat for GA \cc @pruivo

Reply:
yes, the commit of that PR is applied in the internal repository.